fontmaker/core/ttfparser.go

```diff
@@ -224,7 +224,11 @@ func (t *TTFParser) parse(rd io.Reader) error {
 	if err != nil {
 		return err
 	}
-	//t.cacheFontData = fontdata
+
+	return t.DoParse(fontdata)
+}
+
+func (t *TTFParser) DoParse(fontdata []byte) error {
```

Comment:
Sorry, `DoParse()` was ad-hoc naming during my test. I think it should be `ParseFontData(...)` so the developer will see the three functions which does the same thing but has different interface.